packages/front-end/pages/settings/index.tsx

```diff
@@ -163,13 +173,14 @@ const GeneralSettingsPage = (): React.ReactElement => {
     const enabledVisualEditor =
       !settings?.visualEditorEnabled && value.visualEditorEnabled;
 
-    const transformedOrgSettings = {
+    const transformedOrgSettings: Partial<OrganizationSettings> = {
```

Comment:
Is it safe to only put a partial? Do we risk erasing other properties? Normally puts overwrite a record, and patch is the method used to do partial updates.

Reply:
reverted